src/QsCompiler/CommandLineTool/CompilationTracker.cs

```diff
@@ -28,64 +26,69 @@ private class CompilationTask
             /// Represents the name of the parent compilation task.
             /// </summary>
             public readonly string ParentName;
+
             /// <summary>
             /// Represents the name of the compilation task.
             /// </summary>
             public readonly string Name;
+
             /// <summary>
             /// Contains the UTC datetime when the task started.
             /// </summary>
             public readonly DateTime UtcStart;
+
             /// <summary>
             /// Contains the UTC datetime when the task ended.
             /// </summary>
             public DateTime? UtcEnd;
+
             /// <summary>
             /// Contains the duration of the task in milliseconds.
             /// </summary>
             public long? DurationInMs;
+
             /// <summary>
             /// Stopwatch used to measure the duration of the task.
             /// </summary>
-            private readonly Stopwatch Watch;
+            private readonly Stopwatch watch;
```

Comment:
What is the rule here - private things are lower case, public things are upper case? What about internal things? Is the idea to match this in F#?

Reply:
The convention as I know it has public members as PascalCase and private members as camelCase (sometimes with an underscore prefix). I believe internal members are treated the same a public: https://stackoverflow.com/a/46777930/5399520

Reply:
That's [SA1306](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1306.md). It only applies to private fields. Internal or public fields or private methods start with an uppercase letter.

F# private fields are `let foo = ...`. I think those should also start with a lowercase letter also, like most F# names (with the exception of things that can be seen by C#, which are capitalized - I think that's the rule?).

Reply:
I think it would be good to set it up in a way that people don't need to know whether the source was F# or C#, meaning if we could get public F# fields to be uppercase as well.

Reply:
This naming convention only applies to private fields. By definition, private fields are not visible from other projects, so there are no interop concerns.

Reply:
Sorry, I think I misunderstood you, what I mean is: As far as I know, most/all internal/public C# fields in the compiler were already capitalized, so I think the style change would mainly be limited to private fields. I believe most/all of the public F# fields (`member val`) are already capitalized as well - the only lowercase ones I found were in a private type in a test.

Reply:
sounds good - thanks for checking